### DISCUSSION TITLE
Support for ffmpeg loudness equalization filter

### DISCUSSION THREAD

**Discussion by boblund** [mikebrady/shairport-sync#1886] [Ideas]
Hi,

ffmpeg allows loudness normalization via the [loudnorm filter](https://ffmpeg.org/ffmpeg-filters.html#loudnorm]). Here is an example of its usage
```
ffmpeg -i in.m4a -af loudnorm=I=-16:TP=-1.5:LRA=11 out-normalized.m4a
```
It would be nice if one could configure shairport-sync to use this filter so that all audio is played at approximately the same loudness. One approach would be to allow the loudnorm filter parameters to be set in ```shairport-sync.conf```. Maybe there are other approaches.

Thanks,
Bob

**Comment by mikebrady** [comment]
Thanks for the suggestions.

Just FYI, many apps have facilities built in that implement some of these features, at least partly. I'm thinking of things like the Equaliser and Sound Check setting in Apple Music.
